src/ruby/ext/symengine/extconf.rb

```diff
@@ -0,0 +1,74 @@
+require 'mkmf'
```

Comment:
Checking and linking is done by cmake easily and it generates the Makefile for us. We don't need another Makefile generated by `extconf.rb`